src/main.py

```diff
@@ -87,8 +87,12 @@ def flush(self):
     # We can't use symlinks as at least some Android devices have the user storage
     # and app data directories on different mount points.
     os.environ['KOLIBRI_STATIC_USE_SYMLINKS'] = "False"
+elif sys.platform == "darwin":
+    os.environ["KOLIBRI_INSTALLATION_TYPE"] = "mac"
+    os.environ["KOLIBRI_INSTALLER_VERSION"] = "0.1.3"
 
 
+LOCALHOST = 'http://localhost'
 KOLIBRI_ROOT_URL = 'http://localhost:{}'.format(KOLIBRI_PORT)
```

Comment:
Maybe there's a reason not to, but could have this one be derived from LOCALHOST, too.